movo_common/movo_ros/src/movo_joint_interface/helpers.py

```diff
@@ -47,12 +48,19 @@ def dottedQuadToNum(ip):
 
 def get_ip_address(ifname):
     s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
-    return socket.inet_ntoa(fcntl.ioctl(
+    struct_pack = struct.pack('256s', bytes(ifname[:15], encoding='utf-8'))
+    fctnl_ioctl = fcntl.ioctl(
         s.fileno(),
         0x8915,  # SIOCGIFADDR
-        struct.pack('256s', ifname[:15])
-    )[20:24])
-    
+        struct_pack
+    )
+    skt_inet_ntoa = socket.inet_ntoa(fctnl_ioctl[20:24])
+    if str(skt_inet_ntoa) == "10.66.171.1":
+        return os.environ["KINOVA_MOVO1_IP"]
+    if str(skt_inet_ntoa) == "10.66.171.2":
+        return os.environ["KINOVA_MOVO2_IP"]
+    return False
```

Comment:
why not just return skt_inet_ntoa?
isnt os.environ["KINOVA_MOVO1_IP"] == "10.66.171.1" ? 
also os.environ["KINOVA_MOVO2_IP"] == "10.66.171.2"?

I